tools/shared.py

```diff
@@ -931,8 +935,10 @@ def get_llvm_target():
 
 if LLVM_TARGET == WASM_TARGET:
   # wasm target does not automatically define emscripten stuff, so do it here.
-  COMPILER_OPTS = COMPILER_OPTS + ['-DEMSCRIPTEN',
-                                   '-D__EMSCRIPTEN__']
+  COMPILER_OPTS = COMPILER_OPTS + ['-D__EMSCRIPTEN__']
```

Comment:
I'm not sure, the machinery touches the old `EMCONFIGURE_JS` hacks that are used to support the `./configure` architecture. Looks like that mode does not have `__EMSCRIPTEN__` set, as it shouldn't(?).